layers/meta-balena-up-board/recipes-core/images/balena-image.inc

```diff
@@ -14,6 +14,3 @@ BALENA_BOOT_PARTITION_FILES = " \
     grub-efi-bootx64.efi:/EFI/BOOT/bootx64.efi \
     grubenv:/EFI/BOOT/grubenv \
     "
-
-# add to rootfs the firmware for Ampak AP6214a and AP6355 Wireless/Bluetooth
-IMAGE_INSTALL:append += " ampak-firmware"
```

Comment:
The image we produce supports Up Board, Up Core, Up Squared, Up Core Plus. IT's just 1 OS image so removing the wifi firmware will render it broken for the hw which uses it.
Why remove it in the first place?

Reply:
Currently causes following build error:
`ERROR: QA Issue: ampak-firmware: Files/directories were installed but not shipped in any package`

Reply:
Seems to be caused by deprecated override syntax upstream [balena-up-board](https://github.com/AaeonCM/meta-up-board/blob/5573c004c3f3bd31c3e5520861730f0d144d2b2d/recipes-firmware/ampak-firmware/ampak-firmware_1.2.bb#L12)

Reply:
fixed upstream